pkg/framework/plugins/podlifetime/types.go

```diff
@@ -25,10 +25,10 @@ import (
 type PodLifeTimeArgs struct {
 	metav1.TypeMeta `json:",inline"`
 
-	Namespaces                   *api.Namespaces       `json:"namespaces"`
-	LabelSelector                *metav1.LabelSelector `json:"labelSelector"`
-	MaxPodLifeTimeSeconds        *uint                 `json:"maxPodLifeTimeSeconds"`
-	States                       []string              `json:"states"`
-	IncludingInitContainers      bool                  `json:"includingInitContainers"`
-	IncludingEphemeralContainers bool                  `json:"includingEphemeralContainers"`
+	Namespaces                   *api.Namespaces       `json:"namespaces,omitempty"`
```

Comment:
is there anywhere we're specifically making a null check on pointer fields like this?

Reply:
Every plugin validation code is doing some validation. E.g. https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/framework/plugins/podlifetime/validation.go#L37.
Also, each plugin is expected to check for the nil pointer. E.g. https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/framework/plugins/podlifetime/pod_lifetime.go#L54-L57.